modules/nf-core/mashmap/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process MASHMAP"
+    script "../main.nf"
+    process "MASHMAP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mashmap"
+
+    test("Should align the contigs to the genome") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true),
+                ]
+                input[1] = file(params.test_data['sarscov2']['illumina']['contigs_fasta'], checkIfExists: true)
```

Comment:
Is this the right way round? You are passing `genome_fasta` to `-q` and `contigs_fasta` to `-r`?

Can you also update the [paths to the newer format](https://nf-co.re/docs/guidelines/components/modules#input-data) please.

Reply:
Indeed, the order was incorrect. I've adjusted that